vnext/Desktop.IntegrationTests/WebSocketJSExecutorIntegrationTest.cpp

```diff
@@ -19,6 +19,8 @@ using std::move;
 using std::string;
 using std::unique_ptr;
 
+// None of these tests are runnable
+#if 0
 TEST_CLASS (WebSocketJSExecutorIntegrationTest) {
```

Comment:
Wish I had seen this review before.
There are more orthodox ways to disable these tests.
We should aim to run them in UWP too, actually.

Reply:
Let's discuss it offline.

---
In reply to: [434168343](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r434168343) [](ancestors = 434168343)